rosdep/base.yaml

```
@@ -5722,6 +5722,22 @@ libsensors4-dev:
 libserial-dev:
   debian: [libserial-dev]
   ubuntu: [libserial-dev]
+libshaderc-dev:
+  alpine: [shaderc-dev]
+  arch: [shaderc]
+  debian:
+    '*': [libshaderc-dev]
+    bullseye: null
+  fedora: [libshaderc-devel]
+  gentoo: [media-libs/shaderc]
+  opensuse: [shaderc-devel]
+  rhel:
+    '*': [libshaderc-devel]
+    '8': null
+  ubuntu:
+    '*': [libshaderc-dev]
+    'focal': null
+    'jammy': null
```

```suggestion
    focal: null
    jammy: null
```

(I'm nitpicking now, but we don't usually use the quotes here)

---

Removed in https://github.com/ros/rosdistro/pull/40283/commits/2487939499fb175dac9d134f1b285b7dc9560f71. I saw the `'8'` for rhel and didn't think to look at other cases for ubuntu.

---

> Removed in [2487939](https://github.com/ros/rosdistro/commit/2487939499fb175dac9d134f1b285b7dc9560f71). I saw the `'8'` for rhel and didn't think to look at other cases for ubuntu.

Yeah.  It's weird in that you *have* to quote `8` and `*`, but it is optional for strings that start with regular ASCII characters.